packages/CollapsibleText/src/CollapsibleText.js

```diff
@@ -81,7 +81,7 @@ function CollapsibleText(props) {
             aria-controls={contentId}
             aria-expanded={!isCollapsed}
             isSemantic={false}
```

Comment:
This package lacks of changelog, don't remember the CollapsibleText component 🤔 why do we have a collapsible only for text? (rethorical question)